lib/index.js

```diff
@@ -21,6 +21,9 @@ export default class PieClientSideController{
       });
   }
 
+  getLanguages() {
+    return _.intersection.apply(this, _.map(this._pies, (pie) => { return pie.pie.langs }));
```

Comment:
a slightly more lodashy way? `_(this._pies).map('pie.langs').flatten().intersection().value()`